docs/source/conf.py

```diff
@@ -360,7 +362,8 @@ def _does_not_start_with_underscore(input_string):
             + "</a>"
         )
 
-        record = pd.DataFrame([modname, algorithm_type, author_info], index=COLNAMES).T
+        record_cells = [modname, algorithm_type, author_info, maintainer_info]
+        record = pd.DataFrame(record_cells, index=COLNAMES).T
```

Comment:
This line does look a bit weird. If I understand this correctly, may be you can use this type of syntax?

```pycon
>>> 
>>> import pandas
>>> 
>>> row_names = ["r1", "r2"]
>>> column_names = ["c1", "c2", "c3"]
>>> 
>>> cell_values = [["r1c1", "r1c2", "r1c3"], ["r2c1", "r2c2", "r2c3"]]
>>> 
>>> table = pandas.DataFrame(data=cell_values, index=row_names, columns=column_names)
>>> 
>>> table
      c1    c2    c3
r1  r1c1  r1c2  r1c3
r2  r2c1  r2c2  r2c3
>>> 
```

So, you can keep `records` object as list of lists of row data, and create the dataframe once at the end.

Reply:
I see - sure.
It was like that when I found it already.

btw, do you know how to make search and subset in html? Or sorting, as here:
https://en.wikipedia.org/wiki/Help:Sortable_tables#Sorting_buttons_in_a_separate_row

Currently, the "search box" does not work

Reply:
No, sorry. I just know html-css-js is a thing, and that's where my knowledge ends abruptly ;(

But previously @Abhishektharu pointed out this, if you can use it in some way.

Reply:
> But previously @Abhishektharu pointed out this, if you can use it in some way.

did you mea to add a link?

Reply:
I linked to the [original post](https://github.com/sktime/sktime/issues/5495#issuecomment-1786648195), don't know why you can't see.

> I found that the feature (search the table ) worked in v0.20.0 and previous but v0.20.1 and later caused problem. In console i found that (Uncaught ReferenceError: $ is not defined at estimator_overview.html:1992:9) this mainly happens when we don't include jQuery is not properly loaded or initialized in HTML and it is missing after v0.20.0. But it was included in v0.20.0. ![Screenshot 2023-10-31 131856](https://private-user-images.githubusercontent.com/51353634/279293160-0d5b3176-9b38-41ba-aef4-5b5a57a9b4bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc0MTg5ODksIm5iZiI6MTcwNzQxODY4OSwicGF0aCI6Ii81MTM1MzYzNC8yNzkyOTMxNjAtMGQ1YjMxNzYtOWIzOC00MWJhLWFlZjQtNWI1YTU3YTliNGJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA4VDE4NTgwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiMWRiNDFmNzU3ZWZmMDU1OGU1ODBkZjJhYWExNGQwYjUwOWFmYmMzNjdiMTczOWQ4ODNjMmEwMzdjOTQ5NzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UlGU-9MhzlpUy0n6YJP7LYwstJRjhK0a13dFyOSSRfg) **probably including jQuery in head section of html will solve the problem** ![Screenshot 2023-10-31 132037](https://private-user-images.githubusercontent.com/51353634/279293575-9e38a92c-40fc-427b-acf1-6bdcfee279fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc0MTg5ODksIm5iZiI6MTcwNzQxODY4OSwicGF0aCI6Ii81MTM1MzYzNC8yNzkyOTM1NzUtOWUzOGE5MmMtNDBmYy00MjdiLWFjZjEtNmJkY2ZlZTI3OWZlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA4VDE4NTgwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2ZGY5ODQ2NTc2MjRiM2FiNzhjY2QzYWJhODJmNzliOGE4MDc5NzU4ZTI4NzUyZmQ2YzczMDIxMjYwYjcwMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.datDOSGp91JhawF3r-CPMBlURlGKVfZ4xmLoWLBTPfU)

Reply:
> I linked to the [original post](https://github.com/sktime/sktime/issues/5495#issuecomment-1786648195), don't know why you can't see.

not in your original post, you added the link by edit